adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/routes/AssetsResource.kt

```diff
@@ -364,10 +364,14 @@ constructor(
 
         @Parameter(name = "jvm_impl", description = "JVM Implementation", required = true)
         @PathParam("jvm_impl")
-        jvm_impl: JvmImpl
+        jvm_impl: JvmImpl,
+
+        @Parameter(name = "vendor", description = OpenApiDocs.VENDOR, required = false)
+        @QueryParam("vendor")
+        vendor: Vendor?
 
     ): List<BinaryAssetView> {
-        val releaseFilter = ReleaseFilter(ReleaseType.ga, featureVersion = version, vendor = Vendor.adoptopenjdk)
+        val releaseFilter = ReleaseFilter(ReleaseType.ga, featureVersion = version, vendor = vendor)
```

Comment:
I think you need to detect a null vendor and default it to Vendor.adoptopenjdk

Reply:
It already defaults to adoptopenjdk in the [Vendor](https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/adoptopenjdk-api-v3-models/src/main/kotlin/net/adoptopenjdk/api/v3/models/Vendor.kt#L6) class

Reply:
so in the Vendor class we set defaultValue="adoptopenjdk" however I believe that is purely documentation, nothing is done by the framework to enforce it, we have to enforce it ourselves. Doing some local testing it seems that unless specified we get a null vendor

Reply:
Fair enough. Have updated